### ISSUE TITLE
Thank you for this beatiful theme!

### ISSUE THREAD

**Issue by alxgvr** [horst3180/arc-icon-theme#25]
Just wanted to say thank you for this theme!
I tried many icon packs, and Arc feels best for me now.

Shade of blue for folder icons are brilliant! Don`t change it anymore (except new themes wit new colors like gray or green).
Folder icon design is fantastic too. 

I, personally mix it with Super-Ultra-Flat-Numix-Remix theme for apps icons (I don`t like Moka).

**Comment by ghost** [comment]
> Super-Ultra-Flat-Numix-Remix

_What a looooong name :stuck_out_tongue_closed_eyes:_

**Comment by alxgvr** [comment]
Yes, but it looks great) Especially apps icons. I find it better then
Paper. Chek it out: https://github.com/erikdubois/Surfn?files=1

чт, 6 окт. 2016 г., 5:30 Federico Damián Schonborn <notifications@github.com
